### PR TITLE
[docs] update raspbianos installation guide

### PR DESCRIPTION
**Pull request by Jacob27**
### Details:
 - *update raspbianos installation guide*

### Tickets:
 - *CVS-71846*

### CODE REVIEW

File: docs/install_guides/raspbian-detection-sample.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+   ```sh
+   cmake -DCMAKE_BUILD_TYPE=Release -DCMAKE_CXX_FLAGS="-march=armv7-a" /opt/intel/openvino_2022/samples/cpp
+   ```
+   ```sh
+   make -j2 hello_classifiaction
+   ```
+3. Download the pre-trained squeezenet1.1 image classification model with the Model Downloader or copy it from the host machine:
+   ```sh
+   git clone --depth 1 https://github.com/openvinotoolkit/open_model_zoo
+   cd open_model_zoo/tools/downloader
```
Comment (dkurt) [inline docs/install_guides/raspbian-detection-sample.md:25]:
```bash
cd open_model_zoo/tools/model_tools
```

File: docs/install_guides/raspbian-detection-sample.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
+# Build and Run Object Detection Sample {#raspbian_detection_sample}
+
+@sphinxdirective
+
+.. _raspbian detection sample:
+
+@endsphinxdirective
+
+Follow the next steps to use the pre-trained face detection model using Inference Engine samples from the OpenVINO toolkit.
```
Comment (dkurt) [inline docs/install_guides/raspbian-detection-sample.md:9]:
```
Follow the next steps to use the pre-trained image classification model using Inference Engine samples from the OpenVINO toolkit.
```

File: docs/install_guides/raspbian-detection-sample.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
+# Build and Run Object Detection Sample {#raspbian_detection_sample}
```
Comment (dkurt) [inline docs/install_guides/raspbian-detection-sample.md:1]:
```
# Build and Run Image Classification Sample {#raspbian_classification_sample}
```

File: docs/install_guides/raspbian-detection-sample.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+@sphinxdirective
+
+.. _raspbian detection sample:
```
Comment (dkurt) [inline docs/install_guides/raspbian-detection-sample.md:5]: classification

File: docs/install_guides/raspbian-detection-sample.md (hunk 3 above)
Comment (dkurt) [reply docs/install_guides/raspbian-detection-sample.md:1]: Also need to update everywhere - sample is classification, not detection now

File: docs/install_guides/raspbian-detection-sample.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+   cmake -DCMAKE_BUILD_TYPE=Release -DCMAKE_CXX_FLAGS="-march=armv7-a" /opt/intel/openvino_2022/samples/cpp
+   ```
+   ```sh
+   make -j2 hello_classifiaction
+   ```
+3. Download the pre-trained squeezenet1.1 image classification model with the Model Downloader or copy it from the host machine:
+   ```sh
+   git clone --depth 1 https://github.com/openvinotoolkit/open_model_zoo
+   cd open_model_zoo/tools/downloader
+   python3 -m pip install -r requirements.in
```
Comment (dkurt) [inline docs/install_guides/raspbian-detection-sample.md:26]: add
```
python3 -m pip install --upgrade pip
```
before it

File: docs/install_guides/raspbian-classification-sample.md [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+
+Follow the next steps to use the pre-trained image classification model using Inference Engine samples from the OpenVINO toolkit.
+
+1. Navigate to a directory that you have write access to and create a samples build directory. This example uses a directory named `build`:
+   ```sh
+   mkdir build && cd build
+   ```
+2. Build the Hello Classification Sample:
+   ```sh
+   cmake -DCMAKE_BUILD_TYPE=Release -DCMAKE_CXX_FLAGS="-march=armv7-a" /opt/intel/openvino_2022/samples/cpp
```
Comment (dkurt) [inline docs/install_guides/raspbian-classification-sample.md:17]:
```suggestion
2. Download samples source code from https://github.com/openvinotoolkit/openvino/tree/master/samples/cpp
3. Build the Hello Classification Sample:
   ```sh
   cmake -DCMAKE_BUILD_TYPE=Release -DCMAKE_CXX_FLAGS="-march=armv7-a" /path/to/samples/cpp
```

File: docs/install_guides/raspbian-classification-sample.md [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+   ```
+3. Download the pre-trained squeezenet1.1 image classification model with the Model Downloader or copy it from the host machine:
+   ```sh
+   git clone --depth 1 https://github.com/openvinotoolkit/open_model_zoo
+   cd open_model_zoo/tools/model_tools
+   python3 -m pip install --upgrade pip
+   python3 -m pip install -r requirements.in
+   python3 downloader.py --name squeezenet1.1 
+   ```
+4. Run the sample specifying the model, a path to the input image, and the VPU required to run with the Raspbian* OS:
```
Comment (dkurt) [inline docs/install_guides/raspbian-classification-sample.md:30]:
```suggestion
2. Download samples source code from https://github.com/openvinotoolkit/openvino/tree/master/samples/cpp
3. Build the Hello Classification Sample:
   ```sh
   cmake -DCMAKE_BUILD_TYPE=Release -DCMAKE_CXX_FLAGS="-march=armv7-a" /path/to/samples/cpp
   ```
   ```sh
   make -j2 hello_classification
   ```
4. Download the pre-trained squeezenet1.1 image classification model with the Model Downloader or copy it from the host machine:
   ```sh
   git clone --depth 1 https://github.com/openvinotoolkit/open_model_zoo
   cd open_model_zoo/tools/model_tools
   python3 -m pip install --upgrade pip
   python3 -m pip install -r requirements.in
   python3 downloader.py --name squeezenet1.1 
   ```
5. Run the sample specifying the model, a path to the input image, and the VPU required to run with the Raspbian* OS:
```

**Review by dkurt — APPROVED** [review]

**Comment by Jacob27** [comment]
http://openvino-docs.inn.intel.com/jacek-rpi-installation-guide-1/openvino_docs_install_guides_installing_openvino_raspbian.html

**Review by andrew-zaytsev — APPROVED** [review]

**Pull request merged**